src/document/Document.java

```diff
@@ -127,14 +145,18 @@ public String getText()
 		return this.text;
 	}
 	
-	/** return the Flesch readability score of this document */
+	/** A method for calculating the Flesch readability score of this document
+	 * @return Flesch Score based on the mathematical formula
+	 * */
 	public double getFleschScore()
 	{
 	    // TODO: You will play with this method in week 1, and 
 		// then implement it in week 2
-	    return 0.0;
+		int numWords = getNumWords();
+		int numSentences = getNumSentences();
+		int numSyllables = getNumSyllables();
+		double numWordsDivNumSentences = numSentences == 0 ? 0 : ((double)numWords)/numSentences;
+		double numSyllablesDivNumWords = numWords == 0 ? 0 : ((double)numSyllables)/numWords;

```

Comment:
Coding style: `((double) numWords) / numSentences`

Reply:
going to update this PR with the correct portion of the file, but noted on making this update as well!